clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/SubmitJobRequest.java

```diff
@@ -56,6 +58,13 @@ public String getConstraint() {
     }
   }
 
+  @Data
+  public static class Constraints {
+
+    public Map hardConstraints;
```

Comment:
thoughts on mapping to just `hard` and `soft` 

```
constraints: {
  hard: {},
  soft: {}
}  
```

vs

```
constraints: {
  hardConstraints: {},
  softConstraints: {}
}  

```

Reply:
that looks better , i will update it